peekingduck/configs/input/live.yml

```diff
@@ -12,3 +12,4 @@ input_source: 0    # 0 for webcam, rtsp URL for CCTV
 mirror_image: False
 frames_log_freq: 100    # Logs frequency of frames passed in cli
 threading: False    # Enable to speed up FPS. Currently unsupported for Linux.
```

Comment:
1. Didn't default to "True" because I took a conservative stance of enabling the lowest (base) settings and left it to the user to enable threading. We could indicate in a clear note that threading is implemented for use, and default to "True" a few versions later after users have used it and confirmed it works well.
2. Have removed the comment.
3. `buffer_frames` is redundant for `input.live` and will lead to "laggy video" effect. It is designed to go along with `input.recorded` with threading: without buffering, this would lose frames as consumer would be slower than producer (input).
4. I suppose you mean limit the buffer queue size to 30 frames? Alt we could tie `buffer_frames` to `input.recorded` only and force it to `False` for `input.live`.

Reply:
Kept `threading` and `buffer_frames` both `False` by default, but added extensive threading tech notes to highlight the scenarios where they should be enabled and why. Also mentioned `fps_saved_output_video` is to be set as well.

Reply:
Tech notes look good - one final minor comment - I think we can link both `recorded.py` and `live.py` buffer frame config to the tech notes in the `.yml` files if the users need more details?

Should be able to link it as per the example below (do check).
```
    The yolo node is capable of detecting objects from 80 categories. It uses
    YOLOv4-tiny by default and can be changed to using YOLOv4. The table of
    categories can be found :ref:`here <general-object-detection-ids>`.

```

Reply:
Have linked the comments in `live.py` and `recorded.py` to the respective `.yml` files.